code-challenges.test.js

```diff
@@ -36,8 +87,25 @@ const studyMinutesWeek1 = {
   friday: 15,
   saturday: 60
 }
+
 // Expected output: [15, 15, 20, 30, 30, 60, 90]
 
+describe("leastNumber", () => {
+  it("returns an array of the values sorted from least to greatest.", () => {
+    expect(leastNumber(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+  })
+
+})
+// ReferenceError: leastNumber is not defined
+// Pseudocode : 1. Create a function called leastNumber
+//              2. Input is the object of studyMinutesWeek1 sunday through saturday
+//              3.The Object.values returns an array of values of the enumerable properties of an object 
+//              4. Output is the array of the values sorted from least to greatest.
+
```

Comment:
Nice! Very clean code and I like this approach. Can we use a parameter to make this more dynamic? You can be a bit more descriptive with your pseudocode here. For example, how will you sort from least to greatest?